client/containers/Pub/SpubHeader/SpubHeaderTab.tsx

```diff
@@ -18,7 +18,7 @@ const SpubHeaderTab = (props: Props) => {
 	const [minHeight, setMinHeight] = useState(0);
 	const { viewportHeight } = useViewport();
 
-	useEffect(() => {
+	useLayoutEffect(() => {
```

Comment:
I deliberated much when choosing `useEffect` over `useLayoutEffect` and I either never understood or forgot my rationale. Would love to know what changed when you swapped for the latter.